packages/reactotron-redux/src/create-action-tracker.js

```diff
@@ -40,10 +39,24 @@ export default (reactotron, trackerOptions = {}) => {
 
         var unwrappedAction = action.type === 'PERFORM_ACTION' && action.action ? action.action : action
 
-        const exceptionPattern = exceptionPatterns.reduce((prev, pattern) => pattern(unwrappedAction.type) || prev, false);
+        const checkException = (exception, actionType) => {
+          switch (typeof exception) {
+            case 'string':
+              return actionType === exception;
+            case 'function':
+              return exception(actionType);
+            case 'object':
+              return exception.test(actionType);
+              break;
```

Comment:
You can nuke this `break`.  The return just above it will take care of this case.